### PR TITLE
ENT-2911 | update query to filter out refund order with complete status.

### PR DESCRIPTION
**Pull request by HammadAhmadWaqas**
Analytics Pipeline Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] If you have a migration please contact data engineering team before merging.
  - [ ] Before merging run full acceptance tests suite and provide URL for the acceptance tests run.
  - [ ] A member of data engineering team has approved the pull request.

### CODE REVIEW

**Review by moconnell1453 — APPROVED** [review]

File: edx/analytics/tasks/enterprise/enterprise_enrollments.py [hunk 1]
```diff
@@ -16,9 +16,9 @@
 )
 from edx.analytics.tasks.insights.database_imports import (
     ImportAuthUserProfileTask, ImportAuthUserTask, ImportCourseEntitlementTask, ImportCurrentOrderDiscountState,
-    ImportCurrentOrderLineState, ImportCurrentOrderState, ImportEcommerceUser, ImportPersistentCourseGradeTask,
-    ImportProductCatalog, ImportProductCatalogAttributes, ImportProductCatalogAttributeValues,
-    ImportStudentCourseEnrollmentTask
+    ImportCurrentOrderLineState, ImportCurrentOrderState, ImportCurrentRefundRefundLineState, ImportEcommerceUser,
```
Comment (saleem-latif) [inline edx/analytics/tasks/enterprise/enterprise_enrollments.py:19]: `ImportCurrentRefundRefundLineState` Refund is appearing twice in the name is it possible to change this?

**Review by saleem-latif — APPROVED** [review]
just one question otherwise looks good 👍

File: edx/analytics/tasks/enterprise/enterprise_enrollments.py (hunk 1 above)
Comment (HammadAhmadWaqas) [reply edx/analytics/tasks/enterprise/enterprise_enrollments.py:19]: Yes, you are right. I am not sure about renaming this class name, As it is there for last 5 years and we have 5 usages of this class.
What do you suggest in this situation?

Comment (saleem-latif) [reply edx/analytics/tasks/enterprise/enterprise_enrollments.py:19]: I think it is okay if it is difficult to change.

**Review by HassanJaveed84 — APPROVED** [review]

**Pull request merged**